MFaaP.MFWSClient.Tests/MFWSVaultValueListItemOperations.cs

```diff
@@ -149,6 +149,41 @@ public void GetValueListItems_WithLimitAndNameFilter()
 			runner.Verify();
 		}
 
+		/// <summary>
+		/// Ensures that a call to <see cref="MFaaP.MFWSClient.MFWSVaultValueListItemOperations.CreateValueListItemAsync"/>
+		/// requests the correct resource using the correct method.
+		/// </summary>
+		[TestMethod]
+		public async Task  CreateValueListItemAsync()
```

Comment:
Please update the test method to be named "AddValueListItemAsync", as per the name change listed below.